### ISSUE TITLE
Advertised ip ranges resource iteration assigning wrong values to attributes

### ISSUE THREAD

**Issue by dilpreetsingh9** [terraform-google-modules/terraform-google-vpn#92]
### TL;DR

router advertising ip range config is not being rendered properly and failing to deploy terraform code.

### Expected behavior

Expected behaviour is to deploy a router with advertised ip range of 35.199.192.0/19

### Observed behavior

Error: Error creating Router: googleapi: Error 400: Invalid value for field 'resource.bgp.advertisedIpRanges[0].range': 'range'. Must be a CIDR address range., invalid

  on .terraform/modules/vpn/modules/vpn_ha/main.tf line 63, in resource "google_compute_router" "router":
  63: resource "google_compute_router" "router" {

_______________________________________________________________
Plan:
  + router_name       = {
      + bgp                           = [
          + {
              + advertise_mode       = "CUSTOM"
              + advertised_groups    = [
                  + "ALL_SUBNETS",
                ]
              + advertised_ip_ranges = [
                  + {
                      + description = "35.199.192.0/19"
                      + range       = "range"
                    },
                ]
              + asn                  = 64800
              + keepalive_interval   = 20
            },
        ]

### Terraform Configuration

```hcl
router_asn = 64800
  router_name = ""
  router_advertise_config = {
    mode    = "CUSTOM"
    groups = ["ALL_SUBNETS"]
    ip_ranges  =  {
      range = "35.199.192.0/19"
      #description = "Google DNS forward range"
    }
  }
```

### Terraform Version

```sh
0.14.9
```

**Comment by dilpreetsingh9** [comment]
ip_ranges  =  {
     "35.199.192.0/19" = "Google DNS forward range"

___________________
Plan:
              + advertised_ip_ranges = [
                  + {
                      + description = "Google DNS forward range"
                      + range       = "35.199.192.0/19"
                    },
                ]


this configuration worked as per current module code which doesn't make sense.

**Comment by adickinson72** [comment]
Bump
